index.html

```diff
@@ -15,14 +15,17 @@
                 My Logo
             </a>
             <a class="button" href="#">
-                <img src="/ressources/icons/menu-button.png" alt="Nav Button">
+                <img src="ressources/icons/menu-button.png" alt="Nav Button">
             </a>
             <ul class="options">
-                <li><a href="#" class="po">Portafolio</a></li>
-                <li><a href="#" class="po">About</a></li>
-                <li><a href="#" class="po">Contact</a></li>
-                <li><a href="#" class="po"><img src="ressources/icons/Icon-Mail.png" alt="Icon Mail"></a></li>
-            </ul>
+                <button class="close-button"><img src="ressources/icons/Icon - Cancel.png" alt=""></button>
+                <ul class="menuDisplay">
+                    <li><a href="#works" class="po">Portafolio</a></li>
```

Comment:
- [ OPTIONAL ] Please spell `Portfolio ` properly :100:

Reply:
Thank you very much I had not realized that I put it in Spanish :)